app/src/main/res/layout/activity_reminder_edit.xml

```diff
@@ -25,16 +41,17 @@
             android:text="@string/button_create_new"
             android:layout_alignParentBottom="true"
             android:layout_alignParentRight="true"
+            android:layout_alignParentEnd="true"
```

Comment:
Shouldn't alignParentRight be taken out since this was added? Same with alignParentLeft below